images/win/scripts/ImageHelpers/PathHelpers.ps1

```diff
@@ -1,68 +1,51 @@
-function Test-MachinePath{
+function Get-SystemVariable {
```

Comment:
Let's format all functions in a consistent format:

1. Remove [CmdletBinding()]
2. Remove return

```
function Verb-Noun {
    param(
        params
    )

    srcCode
}
```

Reply:
Done

Reply:
Some functions have a new line after param block some not, fix please to use the same style.

Reply:
Oh, I see. Updated.